frontend/js/main.js

```diff
@@ -108,41 +75,26 @@ jQuery(document).on('change', '#upload', function (event) {
  */
 const createEditDataTypeTable = () => {
   globalDataTypeList = JSON.parse(localStorage.getItem('globalDataTypeList'));
-  dataTypeListLength = Object.keys(globalDataTypeList).length;
-  globalDataTypeTable = document.createElement('table');
-  globalDataTypeTable.className = 'data-type-table';
-  globalDataTypeTable.setAttribute('id', 'globalDataType');
-  tableBody = document.createElement('tbody');
-
-  tableRow = document.createElement('tr');
-  tableHeader1 = document.createElement('th');
-  tableHeader1.innerHTML = 'Source';
-  tableHeader2 = document.createElement('th');
-  tableHeader2.innerHTML = 'Spanner';
-  tableRow.appendChild(tableHeader1);
-  tableRow.appendChild(tableHeader2);
-  tableBody.appendChild(tableRow);
+  let dataTypeListLength = Object.keys(globalDataTypeList).length;
+  let tableContent = '';
+  let globalDataTypeTable = '';
+  let globalDataTypeDiv = document.getElementById('globalDataType');
```

Comment:
this element was actually present in schema-conversion.js file .. now I have moved these functions to the schema-conversion.js file since it was related to that component.. and anyways now changed the logic to calling function from html and returning string from function.. similar to setSessionTableContent()

Reply:
Now we will have two elements with same id that is **globalDataType**,
@13shreya please check

Reply:
changed the id attributes accordingly